lib/extractor/oci-archive/layer.ts

```diff
@@ -176,11 +174,17 @@ function getAllManifestsIndexItems(
 ): OciManifestInfo[] {
   const allManifestsInfo: OciManifestInfo[] = [];
   for (const manifest of imageIndex.manifests) {
-    if (manifest.mediaType === "application/vnd.oci.image.manifest.v1+json") {
+    if (
+      manifest.mediaType === "application/vnd.oci.image.manifest.v1+json" ||
```

Comment:
Non-blocking: It may be worth defining constants for these media types for readability.

Reply:
Added constants